src/main/scala/uk/gov/dhsc/htbhf/ClaimSimulation.scala

```diff
@@ -21,11 +21,12 @@ class ClaimSimulation extends Simulation {
   val baseURl = sys.env("BASE_URL")
   val sessionDetailsBaseURl = sys.env("SESSION_DETAILS_BASE_URL")
   var sessionDetailsDomain = new URI(sessionDetailsBaseURl).getHost
-  val numStartUsers = sys.env("PERF_TEST_START_NUMBER_OF_USERS").toInt
-  val numEndUsers = sys.env("PERF_TEST_END_NUMBER_OF_USERS").toInt
-  val soakTestDuration = sys.env("PERF_TEST_SOAK_TEST_DURATION_MINUTES").toInt
-  val responseTimeThresholdFor95thPercentile = sys.env("THRESHOLD_95TH_PERCENTILE_MILLIS").toInt
-  val responseTimeThresholdForMean = sys.env("THRESHOLD_MEAN_MILLIS").toInt
+  val numStartUsers = sys.env.getOrElse("PERF_TEST_START_NUMBER_OF_USERS", "5").toInt
```

Comment:
Can we default to one user at the start?

Reply:
Done. Updated to one in circleci as well